apps/docs/content/frameworks/laravel/env-variables.mdx

```diff
@@ -0,0 +1,434 @@
+:::
+
+### Build Environment Variables using zerops.yml
+
+To set basic env variables for the build environment, add the `envVariables` attribute to the build section in your `zerops.yml`
+
+```yml
+zerops:
+  # define hostname of your service
+  - setup: app
+    build:
+      base: php@8.4
+      os: alpine
+      # OPTIONAL. Defines the env variables for the build environment:
+      envVariables:
+        APP_DEBUG: true
+        APP_ENV: development
+    # ==== how to build your application ====
+    run:
+      base: php-nginx@8.4
+      os: alpine
+      siteConfigPath: site.conf.tmpl
+```
+
+When you need to update a value of an existing env variable or change the set of build or runtime env variables, update your `zerops.yml` and redeploy your application to Zerops.
+
+### Runtime Environment Variables using zerops.yml
+
+To set basic env variables for the runtime environment, add the `envVariables` attribute to the runtime section in your `zerops.yml`
+
+```yml
+zerops:
+  # define hostname of your service
+  - setup: app
+    build:
+      base: php@8.4
+      os: alpine
+    # ==== how to build your application ====
+    run:
+      base: php-nginx@8.4
+      os: alpine
+      siteConfigPath: site.conf.tmpl
+      # OPTIONAL. Defines the env variables for the runtime environment:
+      envVariables:
+        DB_CONNECTION: pgsql
+        DB_DATABASE: db
+        DB_HOST: db
+        DB_USERNAME: ${db_user}
+        DB_PASSWORD: ${db_password}
+        DB_PORT: 5432
+```
+
+When you need to update a value of an existing env variable or change the set of build or runtime env variables, update your `zerops.yml` and redeploy your application to Zerops.
+
+<Dropdown>
+  <DropdownItem title="Environment Variable Restrictions">
+  Environment variable names must start with a letter or underscore, followed by any combination of letters, numbers, or underscores (matching the pattern `[a-zA-Z_]+[a-zA-Z0-9_]*`).
+
+While keys are case-sensitive, each variable name must be unique within a service, regardless of its case.
+
+For variable values, Zerops only accepts ASCII characters, and end-of-line characters are not permitted.
+
+These restrictions apply uniformly across all environment variable types in Zerops, whether they're secret, build, or runtime variables.
+
+```yml
+APP_NAME
+database_url
+API_KEY_123
+_INTERNAL_VALUE
+```
+  </DropdownItem>
+</Dropdown>
+## Generated Environment Variables
+
+Zerops creates several helper variables when a PHP service is created, e.g. `hostname`, `PATH`. Some helper variables are read-only (`hostname`), others are editable (`PATH`). Generated variables cannot be deleted.
+
+Generated env variables are listed on the **Environment variables** page. Scroll to the **Generated variables** section.
+
+<p align="center">
+  <img
+    src="/img/screenshots/nodejs_generated_variables.png"
+    alt="image"
+    width="100%"
+    height="auto"
+  />
+</p>
+
+## How to read Environment Variables from your PHP app
```

Comment:
I don't understand you rules of capitalizing titles tbh, I thought you were using title case "all major words are capitalized, while minor words are lowercased" or not?